### PR TITLE
CQ-4345916 Capture DOR specific properties for the core components

### PR DESCRIPTION
**Pull request by devgurjar**
@Review : @rismehta @vdua @nitigupt
@dod : done
Added support of dor and added test cases

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: bundles/core/src/test/resources/form/dropdown/exporter-multiselect-dropdown.json [hunk 1]
```diff
@@ -11,7 +11,10 @@
   "properties": {
     "af:layout" : {
       "tooltipVisible": false
-    }
+    },
+    "fd:dor":{
+      "dorExclusion":false,
+      "dorColspan":null}
```
Comment (vdua) [inline bundles/core/src/test/resources/form/dropdown/exporter-multiselect-dropdown.json:17]: `dorcolspan` is not a number ?

**Review by vdua — APPROVED** [review]

File: bundles/core/src/test/resources/form/dropdown/exporter-multiselect-dropdown.json (hunk 1 above)
Comment (devgurjar) [reply bundles/core/src/test/resources/form/dropdown/exporter-multiselect-dropdown.json:17]: It is number but input type is text.

File: bundles/core/src/test/resources/form/panel/exporter-rules-panel.json [hunk 2]
```diff
@@ -1,46 +1,58 @@
...
+        "visible":true,
+        "richText":false,
+        "value":"def"
       },
-      "type": "string",
-      ":type": "core/fd/components/form/textinput/v1/textinput"
+      "properties":{
+        "fd:dor":{
+          "dorExclusion":false,
+          "dorColspan":null
```
Comment (rismehta) [inline bundles/core/src/test/resources/form/panel/exporter-rules-panel.json:29]: Why are we sending this when it is null. We should only include in json if it is not null

Comment (devgurjar) [reply bundles/core/src/test/resources/form/panel/exporter-rules-panel.json:29]: By default in v1, dorExclusion and dorColspan contains no value. It gets value only if we explicitly set from authoring. So I guess this shouldn't be part of json by default.

Comment (rismehta) [reply bundles/core/src/test/resources/form/panel/exporter-rules-panel.json:29]: Irrespective of author/publish view, non-nullable values should only be part of json.

**Pull request closed without merging**